diff --git a/dist/headings.js b/dist/headings.js
index fe3fee32b796f69bd59f3b03df20a5fd1cee3a28..d5182d59ea8f369b3eae1e0687f3beb1cc60040f 100644
--- a/dist/headings.js
+++ b/dist/headings.js
@@ -67,10 +67,17 @@ var anchorTag = (properties) => ({
   properties,
   children: []
 });
-function generateSlug({ children, properties }, title, level, counter) {
+function generateSlug(node, title, level, counter) {
+  const { children, properties } = node;
   const slug = properties.id || (properties.id = toSlug(title, counter));
-  properties.className = properties.className ? `${properties.className} heading` : "heading";
-  children.unshift(anchorTag({ href: `#${slug}`, className: "heading-anchor", ariaLabel: `Permalink for ${title}`, tabIndex: -1 }));
+  if (level === 6) {
+    node.tagName = "a";
+    node.children = [];
+    properties.ariaHidden = "true";
+  } else {
+    properties.className = properties.className ? `${properties.className} heading` : "heading";
+    children.unshift(anchorTag({ href: `#${slug}`, className: "heading-anchor", ariaLabel: `章节： ${title}`, tabIndex: -1 }));
+  }
   return slug;
 }
 export {